### ISSUE TITLE
eSpeak and Say All running sentences together in NVDA 2024.1

### ISSUE THREAD

**Issue by tech10** [nvaccess/nvda#16357]
See issue #15739 for the details, with the following exceptions. A line cannot be spelled to reproduce this issue, and now, it takes a few minutes to begin running sentences together without a pause between them.

**Comment by Adriani90** [comment]
@tech10 I tested with a text which is more than 4 pages long in Microsoft word, Notepad, Google Chrome, Firefox and Edge. I cannot reproduce your problem.
eSpeak is behaving as expected, making pauses between sentences. However, the pause between sentences is shorter compared to one core  but this was always the case and is not specific to NVDA 2024.1.

I set speech rate to 40 and boost is turned on as you adviced.

Can you provide a text or a concrete website where you can reproduce this issue?

**Comment by tech10** [comment]
@Adriani90 You're correct, you wouldn't be able to test it with something 4 pages or slightly longer in Microsoft Word. For me, I needed to be reading text for at least 5 minutes before I encountered this issue. I usually have speech rate at 70 with rate boost enabled during my own reading preferences.

This should definitely be more than long enough to fill any testing requirements, either read in Google Chrome or copied to Notepad in my testing.

https://www.fanfiction.net/s/11577249/1/Magic-of-the-Force

**Comment by Adriani90** [comment]
@tech10 are you still having this issue in NVDA last 64bit alpha?
You can test with the last alpha from this link:
https://download.nvaccess.org/snapshots/alpha/

**Comment by tech10** [comment]
Yes, this issue still exists in alpha 53530,36a2e7d4, the latest available for download as of this reply.
